changelogs/fragments/512-ec2_snapshot_add_check_mode_support.yml.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - ec2_snapshot - add check_mode support
+    (https://github.com/ansible-collections/amazon.aws/pull/512)
```

```suggestion
    (https://github.com/ansible-collections/amazon.aws/pull/512).
```

Very much just a niggle, but is a minor style issue.  https://docs.ansible.com/ansible/latest/community/development_process.html#changelog-fragment-entry-format